atd-toolbox/read_replica_proxy/env_template

```diff
@@ -0,0 +1,2 @@
+# Copy this file to `env` and fill in the value
+DATABASE_SERVER=<hostname of reader instance>
```

Comment:
I was doing that googling that I owed this work this morning, and I tripped over a SO comment that cautioned against using the instance endpoints and instead using the cluster endpoints. Do you think that we should adjust this? I'm still learning about this but wanted to share this. https://stackoverflow.com/a/52996681

Reply:
This worked great. Thank you again Mike! 🙌

Reply:
Awesome, give all credit to the person who replied to that question on SO. They are the real MVP!!! 🏆